upgrade_paths/1.3.0_1.4.0/after/002-plugins.postgresql.json

```diff
@@ -0,0 +1,22 @@
+[
+  [ "tags were added for basic-auth plugin",
```

Comment:
Why is ACL separated from the rest of the plugins?

Also, this file should be renamed to `after/002-custom-entities-tags.postgresql.json`

Reply:
Thanks! Fixed 👍